t/unit-tests/u-example-decorate.c

```diff
@@ -0,0 +1,73 @@
+#define USE_THE_REPOSITORY_VARIABLE
+
+#include "unit-test.h"
+#include "object.h"
+#include "decorate.h"
+#include "repository.h"
+
+struct test_vars {
+	struct object *one, *two, *three;
+	struct decoration n;
+	int decoration_a, decoration_b;
+};
+
+static struct test_vars vars;
+
+void test_example_decorate__add(void)
+{
+	void *ret = add_decoration(&vars.n, vars.one, &vars.decoration_a);
+
```

Comment:
Nit: I'd remove the empty newline here, as it doesn't really help to clarify structure. Also applies to the other tests, except when declaring `void *ret;` without assigning it a value as you do in `__lookup()`.